src/App.js

```diff
@@ -652,26 +660,33 @@ function App() {
 
     scoreContainer.style.marginLeft = `${x}px`;
     concentrationContainer.style.marginRight = `${window.innerWidth - x - CANVAS_WIDTH}px`
+
     const backgroundCtx = background.getContext('2d');
-    
     drawBackground(backgroundCtx, '#D9E121', '#FCEE23');
-    window.onkeydown = onKeyDownFactory(snake);
-    snake.generateToken();
-    beginGameLoop(gameCtx, snake);    
-  });
-
-  
-  return (
-    <article className="App">
-      <h1 className={'title'}>Snakes and Apples</h1>
-      <section className={'subtitle'}>
-        <h2 className={'score-container'}>Score: <span id={'score'}>0</span></h2>
-        <h2 className={'concentration-container'}>Concentration: <span id={'concentration'}>278</span> ppm</h2>
-      </section>
-      <canvas id={'background'} className={'background'} width={CANVAS_WIDTH} height={CANVAS_HEIGHT} />
-      <canvas id={'game'} className={'gameboard'} width={CANVAS_WIDTH} height={CANVAS_HEIGHT}/>
-    </article>
-  );
+
+    window.onkeydown = onKeyDownFactory(this.snake);
+    this.snake.generateToken();
+  }
+
+  startGame() {
+    beginGameLoop(this.gameCtx, this.snake);
+    this.setState({ showingStartScreen: false })
+  }
+
+  render() {
+     return (
```

Comment:
Minor styling nit: should have double space indentation rather than 4 spaced here